AntWars/AIs/AI.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+using AntWars.AIs.Converter.Classes;
+using AntWars.Exception;
+
+namespace AntWars.AIs
+{
+    class AI
+    {
+        private static String CLASS_PLAYERAI = "PlayerAI.AI";
+        private static String METHOD_ANT_TICK = "antTick";
+        private static String METHOD_NEXT_TICK = "nextTick";
+        private static String PROPERTY_GAME = "Game";
+        private static String PROPERTY_PLAYER = "Player";
+
+        private Type playerAI;
+        private object instance;
+
+        public AI(String path, Game game, Player player)
+        {
+            try
+            {
+                String content = System.IO.File.ReadAllText(path);
+
+                // So there is no real way for detecting the use of reflection,
+                // so we searching for the import, that is atleast needed when using the BindingFlag.NonPublic for get/set value for private properties.
+                if(content.Contains("System.Reflection"))
```

Comment:
das ist dann ungefähr in der Ai Klasse von der dll ``this.getType().getProperty("Player", System.Reflection.BindingFlag.NonPublic)`` soweit ich da jetzt nachgelesen hab braucht man das um die privaten attribute auszulesen, dadurch muss man entweder ein ``using System.Reflection`` haben oder so wie oben im beispiel mit dem vollen pfad.